agenta-web/src/components/EvaluationTable/EvaluationTable.tsx

```diff
@@ -50,9 +50,13 @@ const EvaluationTable: React.FC<EvaluationTableProps> = ({
     const variants = evaluation.variants
 
     const variantData = variants.map((variant: Variant) => {
```

Comment:
not related to this PR changes. But maybe we should create a wrapper hook around this logic as it's being repeated in several components imperatively